docs/contributors/git-workflow.md

```diff
@@ -1,26 +1,94 @@
 # Git Workflow
 
-A good workflow for new contributors to follow is listed below:
+The Gutenberg project follows a standard pull request process for contributions. See GitHub's documentation for [additional details about pull requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests).
+
+This documentation is intended to help you get started using git with Gutenberg. Git is a powerful source code management tool, to learn git deeply, check out the [Pro Git book](https://git-scm.com/book/en/v2) available free online under CC BY-NC-SA 3.0 license.
+
+If you are unfamiliar with using git, it is worthwhile to explore and play with it. Try out the [git tutorial](https://git-scm.com/docs/gittutorial) as well as the [git user manual](https://git-scm.com/docs/user-manual) for help getting started.
+
+## Overview
+
+An overview of the process for contributors is:
+
 - Fork the Gutenberg repository.
 - Clone the forked repository.
 - Create a new branch.
 - Make code changes.
+- Confirm tests pass.
 - Commit the code changes within the newly created branch.
 - Push the branch to the forked repository.
-- Submit a Pull Request to the Gutenberg repository.
+- Submit a pull request to the Gutenberg repository.
+
+See the [repository management document](/docs/contributors/repository-management.md) for additional information on how the Gutenberg project uses GitHub.
+
+## Git Workflow Walkthrough
+
+The workflow for code and documentation is the same, since both are managed in GitHub. You can watch a [video walk-through of contributing documentation](https://wordpress.tv/2020/09/02/marcus-kazmierczak-contribute-developer-documentation-to-gutenberg/) and the accompanying [slides for contributing to Gutenberg](https://mkaz.blog/wordpress/contribute-documentation-to-gutenberg/).
+
+
+Here is a visual overview of the Git workflow:
+
+![Visual Overview of Git Workflow](https://developer.wordpress.org/files/2020/09/git-workflow.png)
+
+
+**Step 1**: Go to the Gutenberg repository on GitHub and click Fork. This creates a copy of the main Gutenberg repository to your account.
+
+![Screenshot showing fork button on GitHub](https://developer.wordpress.org/files/2020/09/gutenberg-fork.png)
+
+**Step 2**: Clone your forked repository locally, it is located at: `https://github.com/YOUR-USER-NAME/gutenberg`. Cloning copies all the files to your computer. Open a terminal and run:
+
+```bash
+git clone https://github.com/YOUR-USER-NAME/gutenberg
+```
+
+This will create a directory called `gutenberg` with all the files for the project. It might take a couple of minutes, it is downloading the entire history of the Gutenberg project.
```

Comment:
```suggestion
This will create a directory called `gutenberg` with all the files for the project. It might take a couple of minutes because it is downloading the entire history of the Gutenberg project.
```